types/newrelic/index.d.ts

```diff
@@ -471,6 +491,28 @@ export interface LinkingMetadata {
     hostname: string;
 }
 
+export interface LogEvent {
+    /**
+     * The log message
+     */
+    message: string;
+
+    /**
+     * The log level severity. If this key is missing, it will default to "UNKNOWN"
+     */
+    level?: string;
```

Comment:
```suggestion
    level?: string | undefined;
```
(just to keep in line with used connvetions)